web/pdf_page_view.js

```diff
@@ -52,6 +52,85 @@ import { TextHighlighter } from "./text_highlighter.js";
 import { TextLayerBuilder } from "./text_layer_builder.js";
 import { XfaLayerBuilder } from "./xfa_layer_builder.js";
 
+// TODO: Should we make this an app option, or just always
+// enable the feature?
+const ENABLE_ZOOM_DETAIL = true;
+
+class Recorder {
```

Comment:
Is the intention to remove this class once the PR is ready to land, since the patch is already large and complex enough without this code?

Not to mention that there's a bunch logging that I don't believe is desirable in e.g. production-builds.

Reply:
Yes, I plan to remove the second commit :) It's currently here because it has been very helpful when playing around with the PR, but it's definitely not something for production.